### DISCUSSION TITLE
АРС-МП, РРИ и АСОТП на 717 МСК

### DISCUSSION THREAD

**Discussion by fixinit75** [metrostroi-repo/MetrostroiAddon#208] [General]
Предлагаю в него АРС-МП поставить, в Москве такие есть. По мелочам: возможность выбрать старую АСОТП, возможность заменять АСНП на РРИ.

**Comment by glebqip** [comment]
Для АРС-МП версии нужна дополнительная модель на пульт с лампами АРС-МП, то же самое касается АСОТП и РРИ

**Comment by fixinit75** [comment]
> Для АРС-МП версии нужна дополнительная модель на пульт с лампами АРС-МП, то же самое касается АСОТП и РРИ

Лампы необязательны на АРС-МП, в Пеатниках половина таких машин без них спокойно ездит, но ладно. А вот с АСОТП и РРИ не понял. Есть же модели готовые в руках метростроя.

**Comment by glebqip** [comment]
По АРС-МП зачем тогда выбор, если отличия минимальные и они только в поведении АРС?
РРИ неизвестно как на номерных выглядело кроме того, что на 4 блоке была заглушка.
А то что готовые модели есть не отменяет того факта, что их надо редактировать

**Comment by fixinit75** [comment]
Хорошо, убедительно, что раз МП ставить, то и панельку делать. По поводу РРИ, согласен, инфы мало.

**Comment by fixinit75** [comment]
Кстати, зачем отдельную модель то делать с лампами если текстур пульта достаточно?

**Comment by fixinit75** [comment]
> Для АРС-МП версии нужна дополнительная модель на пульт с лампами АРС-МП, то же самое касается АСОТП и РРИ

Какая конкретно нужна модель для блока с МП? Мб я сделаю

**Comment by fixinit75** [comment]
И всё же, АСОТП старый будет? В чем проблема от того же ежа отцепить его модель? Касаемо РРИ на номере, где-то парочка фоток валалась(или же это был РИУ). Если найду, скину.

**Reply by fixinit75** [reply]
Вот статив с РРИ/РИУ, точно не скажу.
![image](https://user-images.githubusercontent.com/78933401/181607448-f666c41a-55c1-41d9-a498-76d24f821a40.png)
